apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -74,7 +77,8 @@ function TransferCoinPage() {
     );
     const [sendError, setSendError] = useState<string | null>(null);
```

Comment:
we should remove this and use toasts to show send errors when we send the transaction

Reply:
It will simplify things I believe

Reply:
In the other PR https://github.com/MystenLabs/sui/pull/8353